lib/qri/entry_writer.go

```diff
@@ -0,0 +1,65 @@
+package qri
+
+import (
+	"github.com/google/skylark"
+	"github.com/qri-io/skytf/lib"
+	"github.com/qri-io/dataset"
+	"github.com/qri-io/dataset/dsio"
+)
+
+// SkylarkEntryWriter creates a skylark.Value as an EntryWriter
+type SkylarkEntryWriter struct {
```

Comment:
phenom. We should get tests up for this guy soon, but this implementation looks great.